pennylane/interfaces/batch/tensorflow.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains functions for adding the TensorFlow interface
+to a PennyLane Device class.
+"""
+# pylint: disable=too-many-arguments,too-many-branches
+import inspect
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.eager import context
+
+import pennylane as qml
+
+
+def _compute_vjp(dy, jacs):

```

Comment:
I suppose we don't need docstrings for internal functions but we could maybe include a comment describing what it does?

Reply:
```suggestion
def _compute_vjp(dy, jacs):
    # compute the vector-Jacobian product dy @ jac
    # for a list of dy's and Jacobian matrices.
```